floss/results.py

```diff
@@ -189,7 +190,7 @@ class ResultDocument:
 
     @classmethod
     def parse_file(cls, path):
-        return cls.__pydantic_model__.parse_file(path)
+        return cls.parse_file(path)
```

Comment:
this results in an infinite loop:

https://github.com/mandiant/flare-floss/actions/runs/4902927902/jobs/8819760807?pr=720

one solution might be to use the mypy plugin described here: https://docs.pydantic.dev/latest/usage/dataclasses/, would you give that a shot? if it doesn't work, i have an idea for a fallback or two.

Reply:
The mypy puligin  as mentioned in https://docs.pydantic.dev/latest/mypy_plugin/#enabling-the-plugin is already enabled in mypy.ini.

Reply:
besides this everything else looks good

Reply:
lets revert this change, add a `  # type: ignore` to the line, and a comment that explains that this field is guaranteed by the pydantic dataclass.

Reply:
@williballenthin , I have made changes. Please have a look at it.